Build error

@claude 
I am getting this error when building. Please fix and make a pr. 

 > [builder 6/6] RUN npm run build:
0.300 
0.300 > spotify-controller-client@1.0.0 build
0.300 > npx tsc && npx vite build
0.300 
7.207 src/components/ErrorBoundary.tsx(1,8): error TS6133: 'React' is declared but its value is never read.
7.208 src/components/MainApp.tsx(110,21): error TS2345: Argument of type 'string | null' is not assignable to parameter of type 'string'.
7.208   Type 'null' is not assignable to type 'string

--- a/client/src/components/ErrorBoundary.tsx
+++ b/client/src/components/ErrorBoundary.tsx
@@ -1,4 +1,4 @@
-import React, { Component, ErrorInfo, ReactNode } from 'react';
+import { Component, ErrorInfo, ReactNode } from 'react';
 
 interface Props {
   children: ReactNode;

--- a/client/src/components/MainApp.tsx
+++ b/client/src/components/MainApp.tsx
@@ -106,7 +106,7 @@ const MainApp: React.FC = () => {
     if (item.alternatives && typeof item.alternatives[0] === 'object') {
       const alternativeIds = (item.alternatives as Array<{ uri: string }>)
         .map(alt => alt.uri ? alt.uri.split(':')[2] : null)
-        .filter(Boolean);
+        .filter((id): id is string => Boolean(id));
       trackIds.push(...alternativeIds);
     }
     

